lib/internal/test/binding.js

```diff
@@ -5,10 +5,5 @@ process.emitWarning(
   'tracked by any versioning system or deprecation process.',
   'internal/test/binding');
 
-// These exports should be scoped as specifically as possible
-// to avoid exposing APIs because even with that warning and
-// this file being internal people will still try to abuse it.
```

Comment:
Is there any reason to switch from the original idea of limiting the set of exports as much as possible here?

Reply:
This was here for a reason. I'm really kinda -1 on changing it without first exploring alternatives.

Reply:
The reason is to help keep this maintainable. We use process.binding quite a bit in tests and it's far less maintainable to expose each individual property as exports on this object. I understand the reasoning but this approach keeps this test object as simple as possible.

Reply:
For instance, `process.binding()` appears 157 times in test/parallel. Several of those are duplicates, but supporting all of those would mean managing a very large number of exports on `internal/test/binding`

Reply:
I agree with @jasnell. I do not see a different way for doing this, could you come up with a different approach @apapirovski?

Reply:
Ping @apapirovski ... there's really not a scalable way of picking and choosing the exports here so if you have a suggested alternative please let me know.